src/components/Tool/FixEmailAddresses/FixEmailAddressPerson/FixEmailAddressPerson.tsx

```diff
@@ -135,168 +160,238 @@ export const FixEmailAddressPerson: React.FC<FixEmailAddressPersonProps> = ({
+                        </RowWrapper>
+                      </Fragment>
+                    ))}
+                    <RowWrapper item xs={12} sm={6}>
+                      <Box display="flex" justifyContent="space-between" px={2}>
+                        <Box>
+                          <Hidden smUp>
+                            <Typography display="inline">
+                              <strong>{t('Source')}: </strong>
+                            </Typography>
+                          </Hidden>
+                          <Typography display="inline">MPDX</Typography>
+                        </Box>
                       </Box>
-                    </Box>
-                  </RowWrapper>
-                  <RowWrapper item xs={12} sm={6}>
-                    <BoxWithResponsiveBorder
-                      display="flex"
-                      justifyContent="flex-start"
-                      px={2}
-                    >
-                      <EmailValidationForm personId={id} />
-                    </BoxWithResponsiveBorder>
-                  </RowWrapper>
-                </Grid>
-              </EmailAddressListWrapper>
-            </Grid>
-          </PersonCard>
-        </Grid>
-        <Grid item xs={12} md={2}>
-          <Box
-            display="flex"
-            flexDirection="column"
-            style={{ paddingLeft: theme.spacing(1) }}
-          >
-            <ConfirmButtonWrapper>
-              <Button variant="contained" style={{ width: '100%' }}>
-                <ConfirmButtonIcon />
-                {t('Confirm')}
-              </Button>
-            </ConfirmButtonWrapper>
-          </Box>
+                    </RowWrapper>
+                    <RowWrapper item xs={12} sm={6}>
+                      <BoxWithResponsiveBorder
+                        display="flex"
+                        justifyContent="flex-start"
+                        px={2}
+                      >
+                        <EmailValidationForm personId={id} />
+                      </BoxWithResponsiveBorder>
+                    </RowWrapper>
+                  </Grid>
+                </EmailAddressListWrapper>
+              </Grid>
+            </PersonCard>
+          </Grid>
+          <Grid item xs={12} md={2}>
+            <Box
+              display="flex"
+              flexDirection="column"
+              style={{ paddingLeft: theme.spacing(1) }}
+            >
+              <ConfirmButtonWrapper>
+                <Button variant="contained" style={{ width: '100%' }}>
+                  <ConfirmButtonIcon />
+                  {t('Confirm')}
+                </Button>
+              </ConfirmButtonWrapper>
+            </Box>
+          </Grid>
         </Grid>
-      </Grid>
-    </Container>
+      </Container>
+
+      {deleteModalOpen && (
+        <Confirmation
+          title={t('Confirm')}
+          isOpen={true}
+          message={t('Are you sure you wish to delete this email address:')}
```

Comment:
How does this get the actual email address into it as the previous modal had?

Reply:
Fixed